modules/rosa-common-commands.adoc

```diff
@@ -73,10 +84,19 @@ $ rosa create cluster --cluster=<cluster_name> --v <level>
 [id="rosa-version_{context}"]
 == version
 
-Displays the `rosa` version.
+Displays the `rosa` version and checks whether a newer version is available.
 
 .Example
 [source,terminal]
 ----
 $ rosa version [arguments]
 ----
+
+.Example 
+Displayed when a newer version of the `rosa` CLI is available.
+[source,terminal]
+----
+$ rosa version
+1.2.12
```

Comment:
The command and output should be in separate code blocks. 

We usually do this by adding `.Example output` as the title above the second code block that contains the output. Might need to play with the whole thing a bit since right now you have text between the .Example heading and the command block. 

I might even suggst to get rid of that text and the "rosa version" command and just keep it as showing example output. Like:

```
// start of new content
.Example output
[source,terminal]
----
1.2.12
There is a newer release version '1.2.15', please consider updating: https://mirror.openshift.com/pub/openshift-v4/clients/rosa/latest/
----
```

Reply:
Good thought, thankyou :)